src/test/core/analyzer_test.ts

```diff
@@ -176,139 +175,130 @@ suite('Analyzer', () => {
           ['MyNamespace.SubBehavior', 'MyNamespace.SimpleBehavior']);
     });
 
-    test(
-        'creates "missing behavior" warnings on imported documents without elements',
-        async () => {
-          const document = await analyzeDocument(
-              'static/chained-missing-behavior/index.html');
-          const chainedDocument = getOnly(document.getFeatures({
-            kind: 'document',
-            id: 'static/chained-missing-behavior/chained.html',
-            imported: true
-          }));
-          const expectedWarning = {
-            code: 'unknown-polymer-behavior',
-            message:
-                'Unable to resolve behavior `NotFoundBehavior`. Did you import it? Is it annotated with @polymerBehavior?',
-            severity: 1,
-            sourceRange: {
-              end: {column: 55, line: 2},
-              start: {column: 39, line: 2},
-              file: 'static/chained-missing-behavior/chained.html'
-            },
-          };
-          assert.deepEqual(document.getWarnings({imported: false}), []);
-          assert.deepEqual(
-              document.getWarnings({imported: true}).map((w) => w.toJSON()),
-              [expectedWarning]);
-          assert.deepEqual(
-              chainedDocument.getWarnings({imported: false})
-                  .map((w) => w.toJSON()),
-              [expectedWarning]);
-        });
+    testName = `creates "missing behavior" warnings on imported ` +
```

Comment:
I really don't like this `testName` pattern vs. a multi-line `test()` call with string literal argument.

Is that just me? This is mostly personal preference so if I'm alone here I'll survive